jina/resources/executors.requests.Chunk2DocRanker.yml

```diff
@@ -0,0 +1,10 @@
+on:
+  ControlRequest:
+    - !ControlReqDriver {}
+  SearchRequest:
+    - !Chunk2DocRankDriver
+      with:
+        depth_range: [0, 2]  # prune root and 1st level
+        start: 0
+        end: 50
```

Comment:
what are these start and end is about?

Reply:
and is there any reason why you want to promote a specialized executor to `resources` level?